ml/src/main/scala/ai/h2o/sparkling/ml/algos/H2OGridSearch.scala

```diff
@@ -89,6 +89,9 @@ class H2OGridSearch(override val uid: String)
   }
 
   private def prepareHyperParameters(): String = {
+    logWarning(
+      "Starting from version 3.32, hyper-parameter names will use the names correspodning to Sparkling Water parameters" +
```

Comment:
nit: `version 3.32` -> `the version 3.32`

Reply:
👍